client/my-sites/themes/theme-showcase.jsx

```diff
@@ -159,7 +159,20 @@ class ThemeShowcase extends Component {
 
 	scrollToSearchInput = () => {
 		if ( ! this.props.loggedOutComponent && this.scrollRef && this.scrollRef.current ) {
-			this.scrollRef.current.scrollIntoView();
+			// If you are a larger screen where the theme info is displayed horizontally.
+			if ( window.innerWidth > 600 ) {
+				return;
+			}
+			const headerHeight = document.getElementById( 'header' )?.getBoundingClientRect().height;
+			const screenOptionTab = document
+				.getElementById( 'screen-options-tab-id' )
```

Comment:
We could select the first element by classname with `document.querySelector()`. Or we could use `document.getElementsByClassName()` too. Up to you.

Reply:
:+1: